book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_reader :rentals
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
```

Comment:
- [ ]  A method for adding rentals should be present. Link to the example in a previous lesson: [article](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship). Please add it to comply with the project requirements.

Reply:
Thank you for your valuable comment. I added the add_rental method as you can see in the image below:

![method2](https://user-images.githubusercontent.com/60292761/141265705-7fb0e636-3a00-4dba-a106-e3b9ddbd37ea.PNG)